plugins/aea-ledger-solana/aea_ledger_solana/solana.py

```diff
@@ -422,29 +421,9 @@ def _try_send_signed_transaction(
             `raise_on_try`: bool flag specifying whether the method will raise or log on error (used by `try_decorator`)
         :return: tx_digest, if present
         """
-        max_supported_transaction_version: Optional[int] = None
-        if isinstance(tx_signed, dict):
-            max_supported_transaction_version = tx_signed.pop(
-                "max_supported_transaction_version", None
-            )
         stxn = self.deserialize_tx(tx=tx_signed)
         txn_resp = self._api.send_raw_transaction(bytes(stxn.serialize()))
-        retries = 15
-        while True and retries > 0:
-            try:
-                tx_digest = str(txn_resp.value)
-                receipt = self.get_transaction_receipt(
-                    tx_digest,
-                    max_supported_transaction_version=max_supported_transaction_version,
-                )
-                if receipt is None:
-                    raise ValueError("Transaction receipt not found.")
-                break
-            except ValueError:
-                time.sleep(1)
-            retries -= 1
```

Comment:
why are we removing this logic? it would be good to have it here to make sure the tx is settled before we return. This is important since we make multiple txs consecutively which can lead to issues if for example the 2nd tx is executed before the first is through.